File: AsteroidTowerDefence/Assets/Custom Assets/Scripts/Move.cs
using UnityEngine;
using System.Collections;

public class Move : MonoBehaviour {

	// Use this for initialization
	void Start () {
	
	}
	
	// Update is called once per frame
	void Update () {
	transform.position -= new Vector3(0.2f, 0 , 0);	
		
	//transform.position += new Vector3(Random.Range(0,0.2f), (Random.Range(0,0.2f)), 0);
	}
}

File: AsteroidTowerDefence/Assets/Custom Assets/Scripts/Move.js
var speed : float = 5.0;

function Start() 
{
 var randomDirection : Vector3 = new Vector3(Random.Range(-359, 359),Random.Range(-359, 359), 0);
 transform.Rotate(randomDirection);
}
 
function Update()
{
  transform.position += transform.forward * speed * Time.deltaTime;
}